.travis.yml

```diff
@@ -1,12 +1,21 @@
 sudo: false
-
 language: go
 
 go:
-- 1.9.x
-- 1.10.x
+  - 1.9.x
+  - 1.10.x
+
+env:
+  - COMPONENT_PATH="./"
+  - COMPONENT_PATH="./cmd/tsdb"
 
 go_import_path: github.com/prometheus/tsdb
 
+before_install:
+  - cd $COMPONENT_PATH
+
+install:
+  - go get -v -t ./...
+
 script:
-- go test -timeout 5m ./...
+  - make check_license style unused test staticcheck
```

Comment:
yeah lets add only the tests that don't fail and will add more after we fix the problems to keep the PR easy to review.

Reply:
`make style unused test` don't fail so lets add those and than will add `check_license` in another PR and `staticcheck` after that.